src/components/HomepageNavbar/index.tsx

```diff
@@ -9,7 +9,7 @@ const navLinks = [
   },
   {
     label: 'Documentation',
-    href: '/docs/introduction/getting-started',
```

Comment:
We have to update the link in the Conduit readme as well, maybe there's other places.

Reply:
@lovromazgon I'll update the internal links and will add a redirect in case that's linked from elsewhere.